source/braille.py

```diff
@@ -1688,6 +1689,20 @@ def _doCursorMove(self, region):
 		elif self.buffer is self.messageBuffer and keyboardHandler.keyCounter>self._keyCountForLastMessage:
 			self._dismissMessage()
 
+	def _handleProgressBarUpdate(self, obj):

```

Comment:
Please add a comment here (referencing #6862) explaining what this is for; i.e. that it updates a dialog if a progress bar inside it changes, since it's very likely that text in the dialog also changed (e.g. remaining time).